### PR TITLE
Update if-in-a-function.md

### PR DESCRIPTION
**Pull request by crosscontrol**
Setting `x := 1` in the `if` statement will always return positive. `fmt.Println` always adds spaces between operands so no need for a space in the return strings. Also wanted to build on a previous step that accounted for the third case.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/go/intro/if-statements/if-in-a-function.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/if-statements/if-in-a-function.md?ref=b0b23444e0ec37fca3d020b86c124a6603044210)

File: go/intro/if-statements/if-in-a-function.md [hunk 1]
```diff
@@ -37,11 +37,12 @@ func main() {
 // separate function called positiveOrNegative
 // that does the same thing as the above if-else function
 func positiveOrNegative(x int) string {
-
-	if x:=1 ; x >  0 {
-		return " is positive"
+	if x > 0 {
+		return "is positive"
+	} else if x < 0 {
```
Comment (Stefan-Stojanovic) [inline go/intro/if-statements/if-in-a-function.md:42]:
```suggestion
	} if x < 0 {

```

Hey crosscontrol, can you remove the `else`?

The point of this insight was to teach people that they do not have to use `else` when the if is in a separate function.

**Review by Stefan-Stojanovic — COMMENTED** [review]
Hey @crosscontrol, nice catch!

Can you just apply the suggestion and I will merge the PR.

Thank you for your contribution! 💪

**Comment by crosscontrol** [comment]
I apologize about the delay, should be good now! Thanks for all the great work on this course. I'm learning a lot and enjoying it.

**Review by Stefan-Stojanovic — APPROVED** [review]

**Pull request merged**